x-pack/plugins/security_solution/public/resolver/store/actions.ts

```diff
@@ -115,7 +115,7 @@ interface AppReceivedNewExternalProperties {
     /**
      * the `_id` of an ES document. This defines the origin of the Resolver graph.
      */
-    databaseDocumentID?: string;
+    databaseDocumentID: string;
```

Comment:
This is required by `ResolverProps` now should it will always be present and can always be passed to this event.